src/main/java/at/ac/tuwien/kr/alpha/grounder/transformation/NormalizeProgramTransformation.java

```diff
@@ -34,8 +36,6 @@ public NormalProgram apply(InputProgram inputProgram) {
 		NormalProgram retVal = NormalProgram.fromInputProgram(tmpPrg);
 		// Transform intervals - CAUTION - this MUST come before VariableEqualityRemoval!
```

Comment:
I think this comment is no longer true and should be removed.

Reply:
Fixed.